src/Sportalytics.Feed.Application/DTOs/UpdateSportEventDto.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Sportalytics.Feed.Application.DTOs;
+
+public sealed record UpdateSpotEventDto(
```

Comment:
Looks like all your DTOs have the same base, I would move it to base abstract parent

Reply:
Good point) Done)